### PR TITLE
Style/usability enhancement changes

### PR DESCRIPTION
**Pull request by omgmog**
These are a bunch of changes related to #10

There will probably be some more commits on this branch before I'm ready to merge, but I thought I may as well get these changes up.

### CODE REVIEW

File: src/_includes/macros/eventinfo.njk [hunk 1]
```diff
@@ -21,16 +21,20 @@
...
+    {% if event.happened == true %}
     <p>It was ✨ Broadcast Live ✨ on the ✨ Information Superhighway ✨</p>
+    {% else %}
+        {% if event.date | occurred %}
+            <p>Currently live 📣</p>
+        {% endif %}
     {% endif %}
     <div class="videoWrapper">
     <iframe width="1120" height="630" src="{{ event.stream }}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
     </div>
```
Comment (javorszky) [inline src/_includes/macros/eventinfo.njk]: These three have been replaced by https://github.com/jsoxford/jsnotoxfordsite/blob/main/src/_includes/macros/eventinfo.njk#L28

Weird it doesn't pull this is in as a change

**Review by javorszky — CHANGES_REQUESTED** [review]
There's a thing that's weird. Piece of code that should have changed, but didn't, but is also not correct. Left inline comment for it.

File: src/_includes/macros/eventinfo.njk (hunk 1 above)
Comment (omgmog) [reply src/_includes/macros/eventinfo.njk]: I think my branch is just behind `main`, I'll rebase it when I get a moment 
image

Comment (omgmog) [reply src/_includes/macros/eventinfo.njk]: That should now be resolved 
image

**Review by javorszky — APPROVED** [review]

**Pull request merged**